src/lib.rs

```diff
@@ -27,6 +27,9 @@ pub struct TableOptions<'a> {
 
     /// Generates serde::Serialize and serde::Deserialize derive implementations
     use_serde: bool,
+
+    /// Generates the CRUD functions for generated models
+    fns: bool,
```

Comment:
personally i would either name this `crud`, `enable_impls`, `use_impls` or `generate_impls`

Reply:
Well, it is named like that because the internal generator function is also named like that. I would suggest merging this first and then thinking about renaming it consistently everywhere.